### PR TITLE
Hicexplorer

### PR DESCRIPTION
**Pull request by joachimwolff**
Adding help text.

### CODE REVIEW

File: tools/hicexplorer/hicBuildMatrix.xml [hunk 1]
```diff
@@ -134,9 +134,76 @@
...
 
-**What it does**
+Creation of the interaction matrix
+===================================
+
+``hicBuildMatrix`` is creating an interactions matrix based on the two given input sam or bam files with the forward and reverse reads. 
+It groups reads together in `bins` of a user defined size to first, reduce the noise level and with this increase  the validity of the 
+interaction matrix. Second, to reduce the size of the interaction matrix. Usualy used values are 10000 bases to 1 MBases. `hicBuildMatrix` 
+is considering the sites around a given restriction site. 
+
```
Comment (fidelram) [inline tools/hicexplorer/hicBuildMatrix.xml]: I suggest the following help message.

``hicBuildMatrix`` creates a contact matrix based on Hi-C read pairs. It requires two sam or bam files corresponding to the first and second mates of the paired-end H-C reads. The sam and bam files should not be sorted by position.  There are two main options to create the Hi-C contact matrix, either by fixed bin size (eg. 10.000 bp) or by bins of variable restriction fragment size length. ``hicBuildMatrix`` generates a quality control output that can be used to analyze the quality of the Hi-C reads.

File: tools/hicexplorer/hicCorrectMatrix.xml [hunk 2]
```diff
@@ -144,19 +144,72 @@
     </tests>
     <help><![CDATA[
 
-**What it does**
+Matrix correction
+==================
 
-Runs Dekker's iterative correction over a Hi-C matrix.
+| ``hicCorrectMatrix`` runs Dekker's iterative correction over a Hi-C matrix (`Imakaev 2012`_.). For correcting the matrix, 
+| it is important to remove the unassembled scaffolds (e.g. `NT_`) and keep only
```
Comment (fidelram) [inline tools/hicexplorer/hicCorrectMatrix.xml]: it is important to remove the unassembled scaffolds (e.g. `NT_`) ,  mitochondrial DNA and Y chromosome.

File: tools/hicexplorer/hicCorrelate.xml [hunk 3]
```diff
@@ -100,11 +100,55 @@
     </tests>
     <help><![CDATA[
 
-**What it does**
+Matrix correlation
+==================
 
-Computes pairwise correlations between Hi-C matrices data.
+Computes pairwise correlations between Hi-C interaction matrices.
```
Comment (fidelram) [inline tools/hicexplorer/hicCorrelate.xml]: We should try to use 'contact matrices' as this is the most common name used.

**Comment by bgruening** [comment]
Thanks @joachimwolff!

**Pull request merged**